controllers/reps.js

```diff
@@ -1,9 +1,22 @@
+<<<<<<< HEAD
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7840230)
```suggestion
const logReps = require("../models/reps");
```